service/controller/resource/statuscondition/create.go

```diff
@@ -21,12 +21,6 @@ import (
 )
 
 func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
-	if r.provider != label.ProviderAWS {
```

Comment:
why did you remove this?

Reply:
good catch this might have been too much ❤️